modules/tensor_mechanics/test/tests/central_difference/1D/gold/1d_consistent_implicit_out.csv

```diff
@@ -0,0 +1,8007 @@
+time,disp_x
```

Comment:
I realize that this is an explicit solve so we should expect several steps. However ~8000 for a test is going to be painful to maintain. I imagine that we'll see diffs rather easily - you might consider using "interval" in your output so that you only get every 100th output or so.

Reply:
Sounds good. Will do that.